librz/bin/p/bin_nso.c

```diff
@@ -1,18 +1,16 @@
+// SPDX-FileCopyrightText: 2021 Florian Märkl <info@florianmaerkl.de>
 // SPDX-FileCopyrightText: 2017-2018 rkx1209 <rkx1209dev@gmail.com>
 // SPDX-License-Identifier: LGPL-3.0-only
 
-#include <rz_types.h>
-#include <rz_util.h>
-#include <rz_lib.h>
 #include <rz_bin.h>
-#include <rz_io.h>
-#include <rz_cons.h>
 #include "nxo/nxo.h"
 #include <lz4.h>
 
 #define NSO_OFF(x)           rz_offsetof(NSOHeader, x)
 #define NSO_OFFSET_MODMEMOFF rz_offsetof(NXOStart, mod_memoffset)
 
+#define VFILE_NAME_DECOMPRESSED "decompressed"
```

Comment:
maybe `deflated` is better since is shorter?

Reply:
The length does not really matter for this, I think `decompressed` is easier to understand.